app/controllers/campaigns_controller.rb

```diff
@@ -90,7 +90,8 @@ def add_organizer_to_campaign
   end
 
   def is_organizer?
-    current_user.is_a?(CampaignsUsers) && current_user.role == CampaignsUsers::Roles::ORGANIZER_ROLE
+    @campaigns_user = @campaign.campaigns_users.where(user_id: current_user.id).first
```

Comment:
This implicitly assumes there's only one CampaignsUsers per user-campaign pair. Otherwise, `first` might be another role for the same user and campaign.

I suggest something like:
```ruby
@campaign.campaigns_users.where(user_id: current_user.id, role: CampaignsUsers::Roles::ORGANIZER_ROLE).any?
```